lib/src/rules/unsafe_html.dart

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2019, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:analyzer/dart/element/type.dart';
+import 'package:linter/src/analyzer.dart';
+import 'package:linter/src/util/dart_type_utilities.dart';
+
+const _desc = r'Avoid unsafe HTML APIs.';
+
+const _details = r'''
```

Comment:
This could use some more details, IMHO. What should folks do instead?

Reply:
As I mention above, there simply aren't any available alternatives. The lint is provided first to help teams spot unsafe HTML API usage, and review individual violations.

Should I codify that here?

Reply:
There **are** safe ways to do this, right? It's really not "don't set src=", it's more "don't set src= without being careful".

So it's kinda a weird lint